corehq/apps/export/models/new.py

```diff
@@ -1607,6 +1698,36 @@ def _add_to_group_schema(group_schema, path_start, prop, app_id, app_version):
         return schema
 
 
+class SMSExportDataSchema(ExportDataSchema):
+    @property
+    def type(self):
+        return SMS_EXPORT
+
+    @classmethod
+    def schema_version(cls):
+        return SMS_DATA_SCHEMA_VERSION
+
+    @classmethod
+    def _get_inferred_schema(cls, domain, identifier):
+        return None
+
+    @classmethod
+    def _get_current_app_ids_for_domain(cls, domain, app_id):
```

Comment:
I don't think you actually have to implement this. instead of overriding the `_new_from_schema` private method, i would override the `generate_schema_from_builds` method that would just return the schema. then you shouldn't need to touch any of `ExportInstance` functions or overwrite any of these methods. some of these constructs don't make as much sense now that we're not using apps. so the naming of `generate_schema_from_builds` is kind of bad.

i wouldn't be too opposed to splitting out `ExportDataSchema` to `AppDependentExportDataSchema`, however i still think we'd need a split between forms and cases, so i'm not sure how much that is buying us. 

what do you think? overriding just the generate_schema function i think would save us writing a decent amount of code